File: 320/socialnetwork/src/main/java/com/socialnetwork/model/Message.java
package com.socialnetwork.model;

import jakarta.persistence.*;
import jakarta.validation.constraints.Size;

@Entity
@Table(name = "tbl_message", schema = "distcomp")
public class Message extends BaseEntity {

    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "tweet_id", nullable = false)
    private Tweet tweet;

    @Column(name = "content", nullable = false, length = 2048)
    @Size(min = 4, max = 2048)
    private String content;

    public Message() {
        super();
    }

    public Message(Tweet tweet, String content) {
        this.tweet = tweet;
        this.content = content;
    }

    public Tweet getTweet() {
        return tweet;
    }

    public void setTweet(Tweet tweet) {
        this.tweet = tweet;
    }

    public String getContent() {
        return content;
    }

    public void setContent(String content) {
        this.content = content;
    }
}

File: 351004/Kuchko/ServerApp/ServerApp/Models/Entities/Message.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace ServerApp.Models.Entities;

[Table("tbl_message")]
public class Message : BaseEntity
{
    [Column("article_id")] public long ArticleId { get; set; }

    [ForeignKey("ArticleId")] // Указываем связь по внешнему ключу
    public virtual Article Article { get; set; } = null!;

    [Column("content")] public string Content { get; set; } = null!;
}